src/app/views/post/_page/page.ts

```diff
@@ -240,4 +240,11 @@ export default class AppPostPage extends Vue implements LightboxMediaSource {
 		this.lightbox.close();
 		this.lightbox = undefined;
 	}
+
+	@Watch('post.id')
+	onPostIdChange() {
+		if (this.stickerTargetController.model.id !== this.post.id) {
```

Comment:
I think it's fine to always just refresh, even if the model somehow didn't change, right? Or is this if condition there for a reason?

Reply:
It probably makes more sense to do this on any change with the post id - revised